src/wrapped/wrappedlibx11_private.h

```diff
@@ -485,7 +485,7 @@ DATAB(_XInitDisplayLock_fn, 4)
 // _XInitDynamicIM
 // _XInitDynamicOM
 //GO(XInitExtension
-//GO(XInitImage
+GO(XInitImage, iFp)
```

Comment:
XImage contains callback. I'll add the handling of them. It may try to execute ARM code in X86 world if the program directly use the function pointer inside the XImage (with macro for example) insteadof using the regular funciton.

Reply:
Sure enough, when I try to load a song file I get

```
Unimplemented Opcode E5 00 40 A0 E1 60 40 84
```

which looks suspiciously like ARM instructions.

Reply:
Is it working now with may latest commit?

Reply:
No, so maybe it isn't this specific function that's causing the crash.

Reply:
it crashes with the same unimplemented opcode?